packages/laravel/src/Architecture/LazyComponentsResolver.php

```diff
@@ -1,10 +1,10 @@
 <?php
 
-namespace Hybridly\Support;
+namespace Hybridly\Architecture;
 
 use Hybridly\Support\Configuration\Configuration;
 
-final class VueViewFinder
+class LazyComponentsResolver implements ComponentsResolver
 {
     protected const DEFAULT_DEPTH = 20;
```

Comment:
The only thing I would change here is making the depth configurable or maybe add type `int` to deep in `loadModules` and `loadModule` and use that as the `depth` param